client/utils/gatsby/page-loading.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * The function is useful in cases where we want to preload a page
+ * but the link of the page isn't rendered on the screen.
+ * For more details, see https://github.com/freeCodeCamp/freeCodeCamp/pull/55472.
+ */
+export const preloadPage = (path: string | null) => {
+  if (!window.___loader || !path) return;
```

Comment:
Will we know if this stops working? i.e. if migrate to a version of Gatsby that doesn't provide it or away from Gatsby entirely/

Reply:
> Though I'm not sure how I can setup the test with the current conditional as it involves the feature flag and the locale.

It would need mocking, unfortunately.

One thought I just had: would we see this in the analytics? It's not _too_ bad if the prefetching is removed, so it might be okay for it to break briefly.

Reply:
Okay. I'm happy to defer to you two on how to proceed.